components/automate-chef-io/content/docs/on-prem-builder.md

```diff
@@ -231,6 +231,17 @@ Select your Gravatar icon on the top right corner of the Chef Habitat Builder we
 
 Once you are signed in to the Chef Habitat Builder UI, select the **New Origin** button and enter `core` as the name for the origin.
 
+## Access Chef Habitat Builder On-prem With Habitat Command-Line Tools
```

Comment:
I do not love this heading; if you change it, make sure to change the corresponding link as well.

Reply:
I don't love this heading either, but I don't have anything better to offer.